.circleci/config.yml

```diff
@@ -39,27 +39,19 @@ jobs:
 
       - run:
           name: Build
-          command: |
-            ./bin/build-images.sh
-            docker-compose build
+          command: make build
 
-      - run:
-          name: Start once
-          command: |
-            # This will start all the dependencies first
-            docker-compose run web sleep 5
+      # TODO: are these steps even needed if we're testing in another job within the workflow?
```

Comment:
What do you guys think?

Reply:
I understand that the initial idea is to avoid pushing an image on Docker if the integration tests fail on it, but there must be a way to express that in the CircleCI config using two distinct jobs?

Reply:
Got it with https://github.com/mozilla-services/kinto-dist/pull/1953/commits/a41b6e506d9ee6f1fbde4a6598e80d0293b9a639

<img width="892" alt="Screen Shot 2021-12-14 at 3 55 30 PM" src="https://user-images.githubusercontent.com/92693437/146077956-d9c85f41-f48a-4806-bb12-90dc053d4e72.png">